manim/mobject/coordinate_systems.py

```diff
@@ -332,21 +335,16 @@ def get_lines_parallel_to_axis(
         self, axis_parallel_to, axis_perpendicular_to, freq, ratio_faded_lines
     ):
         """Generate a set of lines parallel to an axis.
-
         Parameters
         ----------
         axis_parallel_to : :class:`~.Line`
             The axis with which the lines will be parallel.
-
         axis_perpendicular_to : :class:`~.Line`
             The axis with which the lines will be perpendicular.
-
         ratio_faded_lines : :class:`float`
             The number of faded lines between each non-faded line.
-
         freq : :class:`float`
             Frequency of non-faded lines (number of non-faded lines per graph unit).
-
```

Comment:
@ManimCommunity/docs do we have any guidelines about whitespace inside docstrings?